blogs/data-mover/data-mover-1.3/data-mover-1.3.0-overview.md

```diff
@@ -0,0 +1,230 @@
+<p>
+DM is a collective of modules to finish the data movement, specifically, data upload and data download. The modules may include the data mover controllers to reconcile DUCR/DDCR and the data path to transfer data.</p>
+
+<h3>The Velero Built-in Data Mover (VBDM)</h3>
+<p>
+VBDM is the built-in data mover shipped along with Velero, it includes Velero data mover controllers and Velero generic data path.</p>
+
+<h3>The Node-Agent</h3>
+<p>Node-Agent is an existing Velero module that will be used to host VBDM</p>
+
+<h3>The Exposer</h3>
+<p>Exposer is to expose the snapshot/target volume as a path/device name/endpoint that are recognizable by Velero generic data path. For different snapshot types/snapshot accesses, the Exposer may be different. This isolation guarantees that when we want to support other snapshot types/snapshot accesses, we only need to replace with a new Exposer and keep other components as is.</p>
+
+<h3>Velero Generic Data Path</h3>
+<p>VGDP is the collective of modules that is introduced in Unified Repository design. Velero uses these modules to finish data transmission for various purposes. In includes uploaders and the backup repository</p>
+
+
+
+
+<h3>DataUpload (DUCR) spec</h3>
+<p>
+A Kubernetes CR that acts as the protocol between data mover plugins and data movers
+</p>
+<table>
+<thead>
+<tr>
+<th>Field</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>backupStorageLocation</td>
+<td>BackupStorageLocation is the name of the backup storage location where the
+     backup repository is stored.</td>
+</tr>
+<tr>
+<td>cancel</td>
+<td>Cancel indicates request to cancel the ongoing DataUpload. It can be set
+     when the DataUpload is in InProgress phase</td>
+</tr>
+<tr>
+<td>csiSnapshot</td>
+<td>If SnapshotType is CSI, CSISnapshot provides the information of the CSI
+     snapshot.</td>
+</tr>
+<tr>
+<td>dataMoverConfig</td>
+<td>DataMoverConfig is for data-mover-specific configuration fields.</td>
+</tr>
+<tr>
+<td>datamover</td>
+<td>DataMover specifies the data mover to be used by the backup. If DataMover
+     is "" or "velero", the built-in data mover will be used.</td>
+</tr>
+<tr>
+<td>operationTimeout</td>
+<td>OperationTimeout specifies the time used to wait internal operations,
+     before returning error as timeout.</td>
+</tr>
+<tr>
+<td>snapshotType</td>
+<td>SnapshotType is the type of the snapshot to be backed up. Currently the only valid value is CSI</td>
+</tr>
+<tr>
+<td>sourceNamespace</td>
+<td>SourceNamespace is the original namespace where the volume is backed up
+     from. It is the same namespace for SourcePVC and CSI namespaced objects.</td>
+</tr>
+<tr>
+<td>sourcePVC</td>
+<td>SourcePVC is the name of the PVC which the snapshot is taken for.</td>
+</tr>
+</tbody>
+</table>
+
+<h3>Note: For additional specification information please see the <a href=https://github.com/openshift/oadp-operator/blob/master/docs/API_ref.md>API reference documentation</a>
```

Comment:
IMO we can just add pkg go dev links for API spec.
DUCR spec: https://pkg.go.dev/github.com/vmware-tanzu/velero@v1.12.2/pkg/apis/velero/v2alpha1#DataUpload
DDCR spec: https://pkg.go.dev/github.com/vmware-tanzu/velero@v1.12.2/pkg/apis/velero/v2alpha1#DataDownload

Reply:
added reference links and also added links for status.  thank you